CHANGELOG.rst

```diff
@@ -68,7 +68,7 @@ Thanks to Jan Musílek for the implementation!
   are now deprecated.
 
 - Add :meth:`TempDirectory.as_string`, :meth:`TempDirectory.as_path` and
-  :meth:`TempDirectory.as_local`. :meth:`TempDirectory.getpath` is now deprecated.
+  ``TempDirectory.as_local``. :meth:`TempDirectory.getpath` is now deprecated.
```

Comment:
What's this weird edit to an old changelog entry from 2022? 😕❓

Reply:
`as_local` doesn't exist anymore, so it can't be linked.